### ISSUE TITLE
AsyncSelect doesn't announce 'No Options' message

### ISSUE THREAD

**Issue by kpustakhod** [JedWatson/react-select#5510]
In AsyncSelect 'No options' message is not announced by NVDA, this might confuse user that after typing in no any feedback is get:
<img width="1313" alt="Screenshot 2022-12-09 at 17 31 09" src="https://user-images.githubusercontent.com/16884481/206748565-6013b6ef-a6b2-4229-bb4d-2187086bacb1.png">

In syncronious Select NVDA reads out '0 results available for search' when 'no optoins' gets back on search.
<img width="1313" alt="Screenshot 2022-12-09 at 17 30 04" src="https://user-images.githubusercontent.com/16884481/206748552-67e2afa5-f32a-4c1a-acdd-7cbd88c6aeab.png">

Need to provide AsyncSelect with feedback when search returns no results.
